includes/emails/class-woothemes-sensei-email-teacher-completed-lesson.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) exit; // Exit if accessed directly
+
+if ( ! class_exists( 'WooThemes_Sensei_Email_Teacher_Completed_Lesson' ) ) :
+
+/**
+ * Teacher Completed Lesson
+ *
+ * An email sent to the teacher when one of their students completes a Lesson.
+ *
+ * @class 		WooThemes_Sensei_Email_Teacher_Completed_Lesson
+ * @version		1.9.0
+ * @package		Sensei/Classes/Emails
+ * @author 		WooThemes
+ */
+class WooThemes_Sensei_Email_Teacher_Completed_Lesson {
+
+	var $template;
+	var $subject;
+	var $heading;
+	var $recipient;
+	var $learner;
+	var $teacher;
+
+	/**
+	 * Constructor
+	 *
+	 * @access public
+	 * @return void
+	 */
+	function __construct() {
+		$this->template = 'teacher-completed-lesson';
+		$this->subject = apply_filters( 'sensei_email_subject', sprintf( __( '[%1$s] Your student has completed a lesson', 'woothemes-sensei' ), get_bloginfo( 'name' ) ), $this->template );
+		$this->heading = apply_filters( 'sensei_email_heading', __( 'Your student has completed a lesson', 'woothemes-sensei' ), $this->template );
+	}
+
+	/**
+	 * trigger function.
+	 *
+	 * @access public
+	 * @return void
+	 */
+	function trigger( $learner_id = 0, $lesson_id = 0 ) {
+		global $woothemes_sensei, $sensei_email_data;
```

Comment:
use `Sensei()` and avoid the global. 

I know our old functions are like this but all new code going in must use this :)